viewer/src/index.ts

```diff
@@ -8,3 +8,9 @@ export { createLocalSectorModel, createLocalPointCloudModel } from './datasource
 
 export { SectorNode } from './views/threejs';
 export { createThreeJsSectorNode, createThreeJsPointCloudNode } from './views/threejs';
+export { initializeCesiumSectorScene } from './views/cesiumjs/createCesiumSectorNode';
+
+// @ts-ignore
```

Comment:
Why ts-ignore?

Reply:
I agree. I think this was necessary because of some limitation in import -> export in TypeScript vs ECMAScript. Let us try to remove it and see what fails before merging. If necessary, we should leave a comment about this for later.